### PR TITLE
Update CI and docs

### PR DESCRIPTION
**Pull request by Jakuje**
The PRs for CI are already merged so revert back to the upstream version of the actions. Update also relevancy in meta, readme and ads CI badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # ssh
 ![CI Testing](https://github.com/linux-system-roles/ssh/workflows/tox/badge.svg)
+[![CI Ubuntu](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-ubuntu.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-ubuntu.yml)
+[![CI Ubuntu](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-debian.yml/badge.svg)](https://github.com/linux-system-roles/ssh/actions/workflows/ansible-debian.yml)
```
Comment (richm) [inline README.md:4]: CI Debian

**Comment by richm** [comment]
1 minor nit, otherwise, lgtm

File: README.md (hunk 1 above)
Comment (Jakuje) [reply README.md]: Fixed. Thanks!

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**